### PR TITLE
add MPTelemetry 1.1 plan review

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: telemetry/1.1/README.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+This is a minor release of this specification. This release only focuses on the OpenTelemetry Tracing while the metrics and logging in OpenTelemetry is out of scope.
+The following updates are provided:
+
+* Adopt the latest release of OpenTelemetry 
+* Update plugin dependencies to pull in the fixes for any known CVEs
+* Add more TCKs
+
+
+* https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-telemetry-1.0[MicroProfile Telemetry 1.1 Release Record]
```
Comment (Emily-Jiang) [inline telemetry/1.1/README.adoc:16]:
```suggestion
* https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-telemetry-1.1[MicroProfile Telemetry 1.1 Release Record]
```

Comment (Emily-Jiang) [reply telemetry/1.1/README.adoc]: solved

File: telemetry/1.1/README.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "MicroProfile Telemetry 1.1 (under development)"
+date: 2023-05-15
+summary: "Release for MicroProfile 6.1"
```
Comment (jclingan) [inline telemetry/1.1/README.adoc:4]: We haven't committed to the next release being 6.1. I'd leave the summary blank, the same as Config 3.1.

**Review by jclingan — CHANGES_REQUESTED** [review]
One comment on removing 6.1 platform release target.

**Review by jclingan — APPROVED** [review]

**Pull request merged**